lib/sshutils/server.go

```diff
@@ -576,7 +576,7 @@ func (c *connectionWrapper) Read(b []byte) (int, error) {
 	n, err := c.Conn.Read(buff)
 	if err != nil {
 		// EOF happens quite often, don't pollute the logs with EOF
-		if err != io.EOF {
+		if trace.Unwrap(err) != io.EOF {
```

Comment:
FYI there's `trace.IsEOF(err)` that does this.

Reply:
seems too small to have as a separate function, but ok :)